client/.eslintrc.js

```diff
@@ -77,6 +81,14 @@ module.exports = {
             files: ['*.html'],
             extends: ['plugin:@angular-eslint/template/recommended'],
             rules: {}
-        }
+        }/*,
+        {
+            files: ['*.html'],
+            excludedFiles: ['*inline-template-*.component.html'],
+            extends: ['plugin:prettier/recommended'],
+            rules: {
+                'prettier/prettier': ["error", { parser: 'angular' }]
+            }
+        }*/
```

Comment:
Please remove from this PR and either add a comment to the respective issue or open a PR for it